### ISSUE TITLE
A lightweight data migration plugin ?

### ISSUE THREAD

**Issue by alexisju** [piwik/piwik#18911]
Matomo already provide [an official plugin](https://github.com/matomo-org/plugin-Migration) and some instructions to help data migration, but it still require full server access/control and relatively advanced skills. That finally makes complex data migration.

It could be very useful to develop a plugin that provide a more lightweight way to achieve this goal.

That could be apparented to "sync" to Matomo instances, probably sufficient for small database/websites. 

This could facilitate to move data from a server to another or to aggregate statistics from a particular website to a single Matomo instance, or even to have a kind of day to day backup.

What do you think about this?

**Comment by interpixels** [comment]
Yes, a simple backup/migration plugin would be great. Backing up and migrating should be very easy. Just look at WordPress backup/migration plugins.
